src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0001</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        0 = Beginning of data packet
+                        <br></br>
+                        1 = End of data packet </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 2 - Request GO Device Data Message</h3>
+        <p>This message is used by an IOX which requires vehicle information from the GO device. The GO device responds with a GO Multi-Frame Data (0x27) - Type 2 message.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0002</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Message Version = 2</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 3 - Connect And Send Records</h3>
+        <p>This message requests the GO modem initiate a connection to the server.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0003</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Unused</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 4 - Request VIN Message</h3>
+        <p>An IOX uses this message to request the vehicle's VIN from the GO device. The GO device responds with a GO Multi-frame Data (0x27) - Type 3 message.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0004</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Unused</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 12 - Request Identification Of Go Device, Versions</h3>
+        <p>Supported from protocol version 1.1.</p>
+        <p>Sent from the IOX to the GO device requesting the identification information. The GO device responds with a GO Multi-Frame Data (0x27) - Type 12 message.</p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on